### ISSUE TITLE
WCAG - 1.4.6 Contrast (Enhanced)

### ISSUE THREAD

**Issue by FL209003** [ORT-FIS-202108/proyecto-6-caraballo-lopez-speranza#103]
Con el fin de mejorar el producto y cumplir con los estandares del WCAG proponemos el siguiente enhacement.

1.4.6 Contrast (Enhanced): The visual presentation of text and images of text has a contrast ratio of at least 7:1, except for the following: (Level AAA)

Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 4.5:1;

Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

Plataforma: Web. Todas las versiones

Severidad: Alta

Prioridad: Alta

[Link a como obedecer el estándar](https://www.w3.org/TR/WCAG20/)
